be/src/olap/rowset/vertical_beta_rowset_writer_helper.cpp

```diff
@@ -197,9 +197,7 @@ Status VerticalBetaRowsetWriterHelper::_create_segment_writer(
         return Status::Error<INIT_FAILED>("get fs failed");
     }
     io::FileWriterPtr file_writer;
-    io::FileWriterOptions opts;
-    opts.create_empty_file = false;
-    Status st = fs->create_file(path, &file_writer, &opts);
+    Status st = fs->create_file(path, &file_writer);
```

Comment:
the default behavior of the file writer will create empty file.
But here we should not create empty file, so why remove `opts.create_empty_file = false;`?